README.md

```diff
@@ -18,19 +27,17 @@ DataSqueeze supports two types of compaction
         a. Fetch the source file paths to be compacted from the source path provided.
         b. Perform mapreduce job using the following configuration
             1. Mapper maps records together based on same parent directory and emits parent directory as key.
-            2. Reducer reduces records based on same key but writes data to the target directory provided by the user, retaining
-                the directory structure.
+            2. Reducer reduces records based on same key but writes data to the target directory provided by the user, retaining the directory structure.
 
-2. Inplace Compaction - Performs compaction on the source path. This is not recommended on AWS-S3, since the performance will be terrible.
+2. In-Place Compaction - Performs compaction on the source path. This is not recommended on AWS-S3, since the performance will be terrible.
 
-    Below is a high level summary of the steps that Compaction Utility performs during the course of a typical run for inplace compaction.
+    Below is a high level summary of the steps that Compaction Utility performs during the course of a typical run for in-place compaction.
 
 
         a. Fetch the file paths to be compacted from the source path provided.
         b. Perform mapreduce job using the following configuration
             1. Mapper maps records together based on same parent directory and emits parent directory as key.
-            2. Reducer reduces records based on same key but writes data to the target directory provided by the user, retaining
-                the directory structure.
+            2. Reducer reduces records based on same key but writes data to the target directory provided by the user, retaining the directory structure.
```

Comment:
Format looks off

Reply:
I think the format looks better now than before?

Reply:
I would prefer not adding this in the commit range if the text is not updated.
Essentially, keep the master copy for this as is